homeassistant/components/doorbird/device.py

```diff
@@ -178,14 +178,8 @@ async def async_reset_device_favorites(
     hass: HomeAssistant, door_station: ConfiguredDoorBird
 ) -> None:
     """Handle clearing favorites on device."""
-    await hass.async_add_executor_job(_reset_device_favorites, door_station)
-
-
-def _reset_device_favorites(door_station: ConfiguredDoorBird) -> None:
-    """Handle clearing favorites on device."""
-    # Clear webhooks
     door_bird = door_station.device
-    favorites: dict[str, list[str]] = door_bird.favorites()
+    favorites: dict[str, dict[str, Any]] = await door_bird.favorites()
```

Comment:
This was incorrect typing.  Its now obvious that I added the typing to the lib and a `py.typed` marker file